frontend/src/routes/(authenticated)/org/[org_id]/AddOrgMemberModal.svelte

```diff
@@ -74,6 +80,15 @@
     />
   {/if}
   <OrgRoleSelect bind:value={$form.role} error={errors.role} />
+  <span slot="checkbox">
```

Comment:
For now, this works but there might be a better way of doing this.

Reply:
I took a bit of time to figure out a good solution, take a look at the commit I pushed.